Postman_Sort/Postman_Sort.js

```diff
@@ -0,0 +1,84 @@
+/*
+* JavaScript Implementation for Postman Sort Algorithm
+*/
+
+let temp, max, count, maxdigits = 0, c = 0;
+let t1, t2, k, t, n = 1, p, s;
+
+/* Function to arrange the of sequence having same base */
+function arrange(p, s) {
+    for(var i = p; i < s - 1; i++) {
+        for(var j = i + 1; j < s; j++) {
+            if(arr1[i] > arr1[j]) {
+                // swap array1[i] and array1[j]
```

Comment:
You are swapping numbers at so many places. Please write a common swap function.